File: cpp/lv0/n의_배수_고르기/n의_배수_고르기.cpp
#include <string>
#include <vector>

using namespace std;

// numlist를 돌며, n으로 나머지 연산을 했을 때 결과값이 0인 요소만 남김

vector<int> solution(int n, vector<int> numlist) {
    vector<int> answer;
    for (int i = 0; i < numlist.size(); ++i) {
        if (numlist[i] % n == 0) {
            answer.push_back(numlist[i]);
        }
    }
    return answer;
}

File: go/lv0/n의_배수_고르기/n의_배수_고르기.go
func solution(n int, numlist []int) []int {
    // 여기서 var result []int 로 생성해주면 1번문제가 틀렸다고 나온다 머가 문젠지..
    result := []int{}
    for _, num := range numlist {
        if num % n == 0 {
            result = append(result, num)
        }
    }
    return result
}